appserver/itest-tools/src/test/java/org/glassfish/main/admin/test/tool/StartStopITest.java

```diff
@@ -36,6 +36,6 @@ class StartStopITest {
      */
     @Test
     void asadminGet() {
-        assertThat(ASADMIN.exec("get", "*"), asadminOK());
+        assertThat(ASADMIN.exec("get", "'*'"), asadminOK());
```

Comment:
On Windows work with double quotes, on Linux - no. We need some condition here.

Reply:
Aha, I thought it passed on my Windows, but this test failed to start the domain (unable to expose ports). Can you finish the PR? I can't do more on this machine. I just used the opportunity to try it :)

Reply:
Sure ;)